internal/celutils/utils.go

```diff
@@ -0,0 +1,60 @@
+package celutils
+
+import (
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/common/types/ref"
+)
+
+var (
+	celEnv *cel.Env
+)
+
+func init() {
+	env, err := cel.NewEnv(
+		cel.Variable(CELVariableClaims, cel.MapType(cel.StringType, cel.DynType)),
+		cel.Variable(CELVariableSubSHA256, cel.StringType),
+	)
+
+	if err != nil {
+		panic(err)
```

Comment:
Wanna log it?

Reply:
`panic` will print to stderr and then, if nothing is recovering the panic, abort the program. We don't actually want im-sts to run if we can't spin up a CEL environment.